Makefile

```diff
@@ -17,9 +17,21 @@ start-ajv:
 run: start-ajv
 	poetry run python api.py
 
+.PHONY: clean
+clean: ## Clean the temporary files.
+	rm -rf .ruff_cache
+
 lint: lint-python
 	npm run lint
 
+.PHONY: ruff
+ruff: ## Run ruff linter.
+	poetry run ruff check .
+
+.PHONY: black
```

Comment:
Why do we need this as a separate makefile command? `black` is run as part of `lint-python`.  If someone wants to run `black` on it's own it's basically `poetry run black`, the command in script also excludes irrelevant modules.

Reply:
Team voted, and agreed to keep separate calls for linters

Reply:
If you could only change the comment here to `## Run black code check.` to not confuse that we are formatting anything?

Reply:
Yep - comment updated, and updated the Ruff command comment to match